docs/source/filters.rst

```diff
@@ -5,21 +5,19 @@
 Filters
 =======
 
-Filters in Ansible are from Jinja2, and are used for transforming data inside
-a template expression. Jinja2 ships with many filters as does Ansible and also
-allows users to add their own custom filters.
+Filters in Ansible are from Jinja2, and are used to transform data inside
+a template expression. The templates operate on the Ansible controller, and not
+on the target host. Therefore, filters execute on the controller as they augment
+the data locally.
 
-It should be noted that templates operate on the Ansible controller, not on the
-target host, so filters execute on the controller as they augment the data
-locally.
+Jinja2 ships with many filters as does Ansible, and also allows users to add
+their own custom filters.
 
 The **IBM z/OS core collection** includes filters and their usage in sample
-playbooks.
-
-Unlike collections that can be identified at the top level using the
+playbooks. Unlike collections that can be identified at the top level using the
 collections keyword and then simply accessing modules using module names,
-filters even when included in a collection must always be specified in the
-playbook with their fully qualified name.
+filters must always be specified in the playbook with their fully qualified name
+even when included in a collection .
```

Comment:
Filters must be specified in the playbook with the fqn. Do we have to discuss how to drill down to the module using module names here? it is distracting to read. If the edit doesn't compromise anything material, we can leave the module access out

Reply:
We don't need to discuss how to use modules in this section of the doc, so long as we clearly explain how to use filters. 

The reason I brought up modules here is because I expected that filters would work the same way modules do once the collection keyword is configured, i thought i could just say zos_mouleX and zos_filterY but you can't , there is a JIRA open for this issue and Ansible refused to change the way filters are accessed from a collection, meaning they require a fully qualified name. Modules also require the same if you don't use the collections keyword. 

I am ok with what you have in the edit,  i am also ok if you remove the module reference so long as we don't remove the FQN and how it needs to be applied to use a filter included with this collection.

Reply:
It certainly helps to describe in details. shall we write this as an extended description in the next paragraph? when we resolve the JIRA issue, we can update this description. Will that work?